rpm/SPECS/openresty-openssl.spec

```diff
@@ -38,23 +43,29 @@ Provides C header and static library for OpenResty's OpenSSL library.
 
 %patch0 -p1
 
-
 %build
-./config --prefix=%{openssl_prefix} \
-    no-threads \
-    shared -g
-
+./config \
+  no-threads shared zlib -g \
+  --openssldir=%{openssl_prefix} \
+  -I%{zlib_prefix}/include \
+  -L%{zlib_prefix}/%{_lib} \
```

Comment:
@agentzh Ok, for openssl we need a --libdir to tell it to use "lib" otherwise it seems to be the opposite of wherever you put the other libraries (lib or lib64). I think they were concerned more about it building out of the box for major OS and distributions then people doing special things - but at least they provided an option.